overrides/android-override.json

```diff
@@ -552,6 +552,17 @@
         "clientContentFeatures": {
             "state": "enabled",
             "exceptions": [ ]
+        },
+        "mediaPlaybackRequiresUserGesture": {
+            "state": "disabled",
```

Comment:
This will be enabled in Production when we want to use it (after the ship review, app release).